contracts/ExitHandler.sol

```diff
@@ -173,10 +173,22 @@ contract ExitHandler is DepositHandler {
     exitStake = _exitStake;
   }
 
+  function getExitStake() public view returns (uint256) {
```

Comment:
This changes the contract interface. Before I would call contract.exitStake(), now I have to call contract.getExitStake(). Why does this need to be done? And if we have to do it, we have to do the necessary changes in the node and bridgeUI also.